src/analysis/seminorm.lean

```diff
@@ -294,37 +294,66 @@ structure seminorm (𝕜 : Type*) (E : Type*) [semi_normed_ring 𝕜] [add_group
 
 attribute [nolint doc_blame] seminorm.to_add_group_seminorm
 
-private lemma map_zero.of_smul {𝕜 : Type*} {E : Type*} [semi_normed_ring 𝕜] [add_group E]
-  [smul_with_zero 𝕜 E] {f : E → ℝ} (smul : ∀ (a : 𝕜) (x : E), f (a • x) = ∥a∥ * f x) : f 0 = 0 :=
+section of
+
+section semi_normed_ring
+
+variables [semi_normed_ring 𝕜]
+
+private lemma map_zero.of_smul [add_group E] [smul_with_zero 𝕜 E] {f : E → ℝ}
```

Comment:
In general, we avoid private lemmas completely. You never know what might be useful to someone else! (And also sometimes private lemmas breaks jump to code and docgen)

Reply:
I did not include them in the first place, so I think removing them is not in the scope of this PR.